homeassistant/components/enocean/sensor.py

```diff
@@ -25,34 +28,44 @@
 
 DEFAULT_NAME = "EnOcean sensor"
 
-DEVICE_CLASS_POWER = "powersensor"
+SENSOR_TYPE_HUMIDITY = "humidity"
+SENSOR_TYPE_POWER = "powersensor"
+SENSOR_TYPE_TEMPERATURE = "temperature"
+SENSOR_TYPE_WINDOWHANDLE = "windowhandle"
 
 SENSOR_TYPES = {
-    DEVICE_CLASS_HUMIDITY: {
+    SENSOR_TYPE_HUMIDITY: {
         "name": "Humidity",
         "unit": "%",
         "icon": "mdi:water-percent",
         "class": DEVICE_CLASS_HUMIDITY,
     },
-    DEVICE_CLASS_POWER: {
+    SENSOR_TYPE_POWER: {
         "name": "Power",
         "unit": POWER_WATT,
         "icon": "mdi:power-plug",
         "class": DEVICE_CLASS_POWER,
     },
-    DEVICE_CLASS_TEMPERATURE: {
+    SENSOR_TYPE_TEMPERATURE: {
         "name": "Temperature",
         "unit": TEMP_CELSIUS,
         "icon": "mdi:thermometer",
         "class": DEVICE_CLASS_TEMPERATURE,
     },
+    SENSOR_TYPE_WINDOWHANDLE: {
+        "name": "WindowHandle",
+        "unit": "-",
```

Comment:
This doesn't look like a unit of measurement. We can use `None` if there's no proper unit.

Reply:
done